### PR TITLE
Footer fix

### PR DESCRIPTION
**Pull request by Ghat0tkach**
**Description**

This PR fixes #

**Notes for Reviewers**
Renamed SMP and SMC in footer ,
Added wasm filter-view image 

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: _includes/home-page.html
Comment (thisiskaransgit) [inline _includes/home-page.html:1]: Hey @Ghat0tkach, the code snippet will be replacing the existing design, under Webassembly based Envoy Filters section

File: _data/footer.yml
Comment (thisiskaransgit) [inline _data/footer.yml:1]: Thanks

File: _includes/home-page.html
Comment (Ghat0tkach) [reply _includes/home-page.html:1]: Thank you @thisiskaransgit  , done the required changes
But The only problem is that the image isnt as same width like other images , as a result in smaller screens it looks wierd.
Any solution for this?

Comment (thisiskaransgit) [reply _includes/home-page.html:1]: simple styling update will be required

File: _includes/home-page.html [hunk 1]
```diff
@@ -186,7 +186,15 @@ <h2>Manage data plane intelligence with <span class="special-text">WebAssembly f
 				</a>
 			</div>
 		</div>
+		<div class="image flex">
+		<a style="width: auto;" href="{{ site.baseurl }}/assets/images/filter-view.png" data-fancybox="images"
```
Comment (Chadha93) [inline _includes/home-page.html:190]: @Ghat0tkach Make the image size the same as others and add a link to this https://docs.meshery.io/reference/mesheryctl/filter/view. Write this adjacent to the image - "Displays the contents of a specific filter based on name or id"

Comment (Ghat0tkach) [reply _includes/home-page.html:190]: Thank you for the review @Chadha93 , Will do the same

**Comment by Ghat0tkach** [comment]
Done the required changes , Please have a review

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
I'm sure that we needed to replace the current design of the WASM filters section, if not you would need to include a  heading with special text if we are adding a new features section.

**Comment by vishalvivekm** [comment]
@Ghat0tkach Did we cover this item in previous meeting? if not, Let's discuss it on the websites call on Monday. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by Chadha93** [comment]
> I'm sure that we needed to replace the current design of the WASM filters section, if not you would need to include a heading with special text if we are adding a new features section.

@Ghat0tkach .....

File: _data/footer.yml [hunk 2]
```diff
@@ -39,13 +39,14 @@
...
     new_window: true
-  # - name: Service Mesh Interface
-  #   link: https://meshery.io/smi
-  #   new_window: false 
-  - name: Service Mesh Performance
+
+  - name: Service Mesh Interface
+    link: https://meshery.io/smi
+    new_window: false 
+  - name: Cloud Native Performance
```
Comment (Chadha93) [inline _data/footer.yml:49]: @Ghat0tkach SMI can be commented.

**Review by Chadha93 — APPROVED** [review]

**Pull request merged**